docs/source/purpose.rst

```diff
@@ -26,32 +26,11 @@ of the existing projects. This table was mostly based on the existing documentat
 understand that the field is rapidly evolving, and that this table might reflect the reality. If you notice
 inconsistencies, please let us know by `opening an issue <https://github.com/ivadomed/ivadomed/issues>`_.
 
-General overview
-++++++++++++++++
-
 .. csv-table::
-   :file: general_overview.csv
+   :file: comparison_dl_soft_transposed.csv
```

Comment:
i would simply call the table the way it was before, ie: "comparison_other_projects_table.csv"

Reply:
Done